### DISCUSSION TITLE
"Cannot generate line item for variant \"\" without a price" type :  "invalid_data"

### DISCUSSION THREAD

**Discussion by olalonde** [medusajs/medusa#5210] [Q&A]
I just setup Medusa. Set CAD as the only currency. Added a product and set a price in CAD. Now when I go to the storefront app, I'm getting this error when clicking "add to cart":

"Cannot generate line item for variant \"\" without a price" type :  "invalid_data"

<img width="280" alt="Screenshot 2023-09-26 at 2 04 39 PM" src="https://github.com/medusajs/medusa/assets/416585/8469d6fb-83ad-42fd-93fc-a0bbecbe0599">

Is this a bug or am I doing something wrong?

**Comment by olalonde** [comment]
Figured it out. I had to edit the Regions in admin panel as well.

**Reply by Harshit-verma-25** [reply]
can you tell a bit more specific as im also facing the same problem but unable to fix it.
